### ISSUE TITLE
What is the recommended format of the Audio files used for training?

### ISSUE THREAD

**Issue by CreateTheImaginable** [Stability-AI/stable-audio-tools#14]
What Audio format does Stable Audio use for training?  More importantly what Audio file format is recommended and what are its Audio file settings and/or parameters?

**Comment by CreateTheImaginable** [comment]
@diontimmer great info!  It might be good to post some guidelines to use the WAV file type and to use 16bit lossless WAV files for best compatibility and that the sample rate is configurable.

Is there an optimum sample rate?

I think it would be good to establish a baseline that could maybe possibly be the standard?  I am sure people will play around and do optimizations but it might be good to have a baseline for comparison?

**Comment by diontimmer** [comment]
yes, most modern audio is in either 44100 or 48000 sample-rate (hz). i personally prefer 44100 for the longer length; the quality difference is not noticable to me
